pvHelpers/certificate_bundle.py

```diff
@@ -0,0 +1,114 @@
+import codecs
+import os
+import re
+import ssl
+import subprocess
+import sys
+
+import certifi
+
+from .misc import g_log
+
+
+class CertificateBundle(object):
+    path = None
+
+    def __init__(self, path=None):
+        self.path = path
+
+    @staticmethod
+    def get_pems_win(store_names=('CA', 'ROOT')):
+        certs = []
+        try:
+            ssl_context = ssl.create_default_context()
+            ssl_context.load_default_certs()
+            for der_cert in ssl_context.get_ca_certs(binary_form=True):
+                certs.append(ssl.DER_cert_to_PEM_cert(der_cert))
+        except AttributeError:
+            import wincertstore
+            for store_name in store_names:
+                with wincertstore.CertSystemStore(store_name) as store:
+                    for cert in store.itercerts(usage=wincertstore.SERVER_AUTH):
+                        try:
+                            pem = cert.get_pem()
+                            pem_entry = '# Label: \'{name}\'\n{pem}'.format(
+                                name=cert.get_name(),
+                                pem=pem.decode('ascii') if isinstance(pem, bytes) else pem
+                            )
+                        except UnicodeEncodeError as e:
+                            g_log.exception(e)
+                            pem_entry = ''
+
+                        certs.append(pem_entry)
+
+        return certs
+
+    @staticmethod
+    def get_pems_darwin():
+        certs = []
+        try:
```

Comment:
can we add comments on why the `['/usr/bin/security', 'find-certificate', '-ap']` is needed?
like which store it fetches from, etc.